cmd/neutrond/root.go

```diff
@@ -85,6 +87,15 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+func initAppConfig() (string, interface{}) {
+	srvCfg := serverconfig.DefaultConfig()
+
+	return gaiaparams.CustomConfigTemplate(), gaiaparams.CustomAppConfig{
+		Config:               *srvCfg,
+		BypassMinFeeMsgTypes: app.GetDefaultBypassFeeMessages(),
```

Comment:
No, code in the `root.go` prepares `app.toml` file during init with default settings. So it should contain default value for `bypass-min-fee-msg-types` option, whereas code in the `app.go` reads this config file and in case if appropriate parameter is empty (for example config was manually updated) it will use default value.